managed/services/supervisord/supervisord.go

```diff
@@ -443,15 +443,13 @@ func (s *Service) marshalConfig(tmpl *template.Template, settings *models.Settin
 
 	templateParams["PMMServerHost"] = ""
 	if settings.PMMPublicAddress != "" {
+		if !strings.HasPrefix(settings.PMMPublicAddress, "https://") && !strings.HasPrefix(settings.PMMPublicAddress, "http://") {
+			settings.PMMPublicAddress = fmt.Sprintf("https://%s", settings.PMMPublicAddress)
```

Comment:
Yeah in this case, it doesn't really matter since we only add it for URL parsing to pass. One possible issue is if `settings.PMMPublicAddress` ends up being used somewhere else, but addressing this [comment](https://github.com/percona/pmm/pull/2803#discussion_r1473358397) should fix that.